### DISCUSSION TITLE
Pair more devices

### DISCUSSION THREAD

**Discussion by cutics** [kormax/apple-home-key-reader#21] [Q&A] [answered]
great work, thank you - it just works.
How do i add a 2nd, 3rd, more devices, once 1st device (iphone) has been successfully paired and is up and running?
It seems like pairing mode is not active any longer, once setup is done for 1st device.
Again, Thank you.

**Comment by kormax** [comment, marked as answer]
A device can be paired to one home only. 

To let other people use the same key - you have to invite them to your Apple Home.

**Comment by seppbastel** [comment]
Invite is only possible with a control center (Apple Tv or homePod).  This means that another device is required, which is not good. I
Me and my wife only want to use the lock and not the whole possibility of the HomeApp

**Reply by kormax** [reply]
Sad that this capability was taken from us with the new HomeKit architecture.

**Comment by seppbastel** [comment]
There is no way to integrate two homes.
